### ISSUE TITLE
Disable notification

### ISSUE THREAD

**Issue by kasbah** [adam-p/markdown-here#282]
Is there a way to disable notification? I seem to get the notification every time I open Firefox now so that might be related to #119, either way I would like to disable it altogether actually.

**Comment by Creazynerd** [comment]
No, please.Sorry, it's my github setting problem. If you disable it, maybe other guys will not receive the import message.

**Comment by kasbah** [comment]
I can't really understand you Creazynerd. What is your "github setting problem"? I was just talking about the update notification that the extension makes after it is updated. I don't really want to see it, ever.

**Comment by adam-p** [comment]
@kasbah: Can you report back whether you're actually getting the notification every time? There were two Firefox releases in quick succession lately, so it might have just seemed like every time.

Either way, you're right that there should be an option to disable the notification.

**Comment by kasbah** [comment]
It didn't show up this time around but it did show up quite a bit,  definitely more than twice. Will it actually show up on next start if I don't dismiss it? It may also be related to FF Sync which I use across two laptops and/or the fact that I don't shut down FF properly often (because it doesn't shut down properly when I just press the power button).
